vertical-pod-autoscaler/templates/admission-controller-service.yaml

```diff
@@ -7,6 +7,8 @@ spec:
   ports:
     - port: 443
       targetPort: 8000
+    - port: metrics
```

Comment:
to keep consistency, this should also be toggled based on the service monitor being enabled or not

Reply:
right missed that one